### PR TITLE
add toast error and automatic dismiss

### PR DESCRIPTION
**Pull request by chuyouchia**
This PR adds:
1. more chat prompts
2. better margin padding
3. Toast error when file is not actively selected in the ask file mode, alongside with automatic dismissal
4. also added a discord CTA in the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,9 @@ A self-organizing AI note-taking app that runs models locally.</h4>
 
 </p>
 
+<h2 align="center">
+    New: We are now accessible via discord, hop by to give ❤️feedback❤️ or discuss our upcoming features!
```
Comment (samlhuillier) [inline README.md:17]: Why no link here?

File: src/components/Chat/Chat.tsx [hunk 2]
```diff
@@ -45,6 +46,15 @@ const ChatWithLLM: React.FC<ChatWithLLMProps> = ({ currentFilePath }) => {
     fetchDefaultModel();
   }, []);
 
+  const fileNotSelectedToastId = useRef<string| null>(null);
+  useEffect(() => {
+    if (!currentFilePath && askText === AskOptions.AskFile) {
+      fileNotSelectedToastId.current = toast.error("Please select a file before asking questions in ask file mode", {}) as string;
```
Comment (samlhuillier) [inline src/components/Chat/Chat.tsx:52]: Perhaps please "open" a file would be nicer wording here

File: src/components/Chat/Chat.tsx [hunk 3]
```diff
@@ -20,7 +21,7 @@ enum AskOptions {
 }
 const ASK_OPTIONS = Object.values(AskOptions);
 
-const PROMPT_OPTIONS = ["Generate weekly 1-1 talking points from this file"]; // more options to come
+const PROMPT_OPTIONS = ["Generate weekly 1-1 talking points from this file", "Separate concepts from todos"]; // more options to come
```
Comment (samlhuillier) [inline src/components/Chat/Chat.tsx:24]: Perhaps cursor-pointer would be nice over the prompt suggestion buttons

**Review by samlhuillier — COMMENTED** [review]
Screenshot 2024-03-03 at 22 40 54
Some centering of the text vertically within these buttons would be nice too

File: src/components/Chat/Chat.tsx (hunk 3 above)
Comment (chuyouchia) [reply src/components/Chat/Chat.tsx:24]: I think it already has cursor pointer though?

**Comment by chuyouchia** [comment]
> Screenshot 2024-03-03 at 22 40 54 Some centering of the text vertically within these buttons would be nice too


Centralized it, good catch!
image

File: src/components/Chat/Chat.tsx (hunk 3 above)
Comment (samlhuillier) [reply src/components/Chat/Chat.tsx:24]: Screenshot 2024-03-03 at 23 44 03
hovering here ^ it doesn't

**Review by samlhuillier — APPROVED** [review]
looking good!

**Pull request merged**